src/main/webapp/translation.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Translation</title>
+
+    <script src="/js/translation-loader.js"></script>
+
+    <style>
```

Comment:
Just nitpicking here but placing all styles in css files, even so all of it in the same file for now, would help to the overall organization of the project, but it is not as needed for unguided projects as it is for our future prototype

Reply:
Agree with memo on that. It's good practice, but optional and up to you

Reply:
Done!